Source/Core/InputCommon/ImageOperations.cpp

```diff
@@ -45,7 +45,8 @@ void CopyImageRegion(const ImagePixelData& src, ImagePixelData& dst, const Rect&
 std::optional<ImagePixelData> LoadImage(const std::string& path)
 {
   File::IOFile file;
-  file.Open(path, "rb");
+  if (!file.Open(path, "rb"))
+    return std::nullopt;
   Common::UniqueBuffer<u8> buffer(file.GetSize());
   file.ReadBytes(buffer.data(), file.GetSize());
```

Comment:
```suggestion
  file.Open(path, "rb");
  Common::UniqueBuffer<u8> buffer(file.GetSize());
  if (!file.ReadBytes(buffer.data(), file.GetSize()))
    return std::nullopt;
```
Checking the read result is probably more relevant than the open result.